MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1443,6 +1446,13 @@ public void writeToXml(PrintWriter pw1, int indent) {
                         + "<edge>"
                         + String.valueOf(getOptionList("::", PilotOptions.EDGE_ADVANTAGES))
                         + "</edge>");
+            // For support personnel, write an available edge value
+            if (isSupport() || isEngineer()) {
```

Comment:
Should this check for isAdmin etc too, like above?

Reply:
Doesn't need to. isSupport() covers astechs and medtechs not covered by isTech() and isDoctor() respectively.  Neither can use edge currently, but I didn't mind them having the value.